common_module.py

```diff
@@ -229,9 +229,12 @@ def process_line(re_search, line, re_match_positions):
 parser.add_option("-F", "--ForceNames", action="store_true", dest="FORCE_NAMES", default=False,
                   help="force names to show up instead of numbered WNs even for very small numbers of WNs")
 # parser.add_option("-f", "--setCOLORMAPFILE", action="store", type="string", dest="COLORFILE")
+parser.add_option("-f", "--setCUSTOMCONFFILE", action="store", type="string", dest="CONFFILE")
 parser.add_option("-m", "--noMasking", action="store_true", dest="NOMASKING", default=False,
                   help="Don't mask early empty WNs (default: if the first 30 WNs are unused, counting starts from 31).")
-parser.add_option("-o", "--SetVerticalSeparatorXX", action="store", dest="WN_COLON", default=0,
+parser.add_option("-o", "--option", action="store", dest="OPTION", type="string", default=None,
+                  help="Override respective option in QTOPCONF_YAML file")
+parser.add_option("-V", "--SetVerticalSeparatorXX", action="store", dest="WN_COLON", default=0,
```

Comment:
this was actually an option that inserts a separator every XX columns. But it was put in the conf file anyway.